modules/hyprpaper/hm.nix

```
@@ -0,0 +1,21 @@
+{
+  config,
+  lib,
+  ...
+}: let
+  settings = {
+    preload = ["${config.stylix.image}"];
+    wallpaper = [" ,${config.stylix.image}"];
```

IIRC, the `,` seperator does not require spaces:

```suggestion
    wallpaper = [",${config.stylix.image}"];
```

Please double check this.